.gitignore

```diff
@@ -65,6 +65,8 @@ htmlcov/
 test-reports/
 test-results.xml
 test-output.xml
+homeassistant-*
+homeassistant.egg*
```

Comment:
This should not be part of this PR

Reply:
Not sure how that got in there - probably a bad merge. Will sort out.